lib/internal/Magento/Framework/Mail/Message.php

```diff
@@ -167,6 +167,7 @@ private function createMimeFromString($body, $messageType)
     {
         $part = new Part($body);
         $part->setCharset($this->zendMessage->getEncoding());
+        $part->setEncoding(Mime::ENCODING_QUOTEDPRINTABLE);
```

Comment:
```suggestion
        $part->setEncoding(Mime::ENCODING_QUOTEDPRINTABLE);
        $part->setDisposition(Mime::DISPOSITION_INLINE);
```
I agree with your [comment](https://github.com/magento/magento2/pull/23649#issuecomment-510057282), adding the disposition to match the previous [ZF1 implementation](https://github.com/magento/zf1/blob/master/library/Zend/Mail.php#L413) should be done.

Reply:
Added as requested